week-1-practise/level-3/1-piping.js

```diff
@@ -16,16 +16,18 @@ var startingValue = 5;
 /* --------------------------- */
 
 function add(a, b) {
+  b = 10;
   return a + b;
 }
 
 function multiply(a, b) {
+  b = 2;
```

Comment:
See my comment above. 
You have changed the definition of these functions, but you are not calling them, which means they are not running.

Reply:
Have a look here and see the difference between *defining* what a function should do (this never changes), and *calling* a function (= using it to do something).